amida-tech/orange-api
# self sufficient test scripts

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Adds .env.test for use in testing environment 
Creates database on testing and starting of the app.
Fixes issue with tests not working the first time
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/SER-225
https://jira.amida-tech.com/browse/SER-222
#### How should this be manually tested?
run `cp .env .env.test`
run `yarn check_db_test`
Make sure it FAILS
change the db name in `.env.test` file
run `yarn test` verify it works

#### Any background context you want to provide?
This does not have a database creation script due to how mongo works. instead the file `check_db_test.js` is implemented so it works similarly to the other services. 
#### Screenshots (if appropriate):
